torch/export/exported_program.py

```diff
@@ -681,6 +677,197 @@ def _decompose_exported_program(
     return exported_program
 
 
+class CustomDecompTable(Dict[torch._ops.OperatorBase, Callable]):
```

Comment:
maybe this class implementation should live in a private file, so it isn't part of the `torch.export.exported_program` public module?